### ISSUE TITLE
bfloat16 and int8_bfloat16

### ISSUE THREAD

**Issue by Purfview** [guillaumekln/faster-whisper#379]
Can bfloat16 and int8_bfloat16 compute types be used by faster-whisper?

**Comment by guillaumekln** [comment]
Yes, but I haven't tested these types with Whisper models. They should work, but they should not be faster than float16 and int8_float16.

**Comment by Purfview** [comment]
I already have them added to cli options, just making sure as the readme isn't updated yet.

By quick look at Wikipedia, I guess it should reduce memory requirement or model size!?

**Comment by guillaumekln** [comment]
No the memory usage and model size are the same as float16 (or int8_float16). But bfloat16 can represent a larger range of values which is required by some model.
